exonum-java-binding-core/rust/src/utils/path.rs

```diff
@@ -0,0 +1,40 @@
+use std::env;
+use std::path::PathBuf;
+
+#[cfg(windows)]
+pub const PATH_SEPARATOR: &str = ";";
+#[cfg(not(windows))]
+pub const PATH_SEPARATOR: &str = ":";
+
+/// Joins several classpaths into a single classpath, using the default path separator.
+/// Preserves the relative order of class path entries.
+pub fn join_paths(parts: &[&str]) -> String {
+    parts.join(PATH_SEPARATOR)
+}
+
+/// Returns current directory (where executable is placed).
+pub fn current_directory() -> PathBuf {
```

Comment:
Isn't 'current' easy to confuse with 'working' directory? I'd consider 'installation' or 'origin' or 'executable'.